Kickstarter-iOS/Views/Cells/SearchProjectCell.swift

```diff
@@ -36,61 +33,44 @@ internal final class SearchProjectCell: UITableViewCell, ValueCell {
+      |> UIImageView.lens.tintColor .~ .white
 
-    _ = self.columnsStackView
-      |> UIStackView.lens.alignment .~ .top
-      |> UIStackView.lens.spacing %~~ { _, stackView in
-        stackView.traitCollection.isRegularRegular
-          ? Styles.grid(4)
-          : Styles.grid(2)
-      }
-      |> UIStackView.lens.layoutMarginsRelativeArrangement .~ true
-      |> UIStackView.lens.layoutMargins .~ .init(topBottom: 0, leftRight: Styles.grid(2))
+    _ = self.metadataLabel
+      |> UILabel.lens.textColor .~ .white
+      |> UILabel.lens.font .~ .ksr_headline(size: 12)
 
-    _ = self.imageShadowView |> dropShadowStyle()
-
-    _ = self.projectImageView
-      |> UIImageView.lens.contentMode .~ .scaleAspectFill
-      |> UIImageView.lens.clipsToBounds .~ true
-
-    self.projectImageWidthConstraint.constant = self.traitCollection.isRegularRegular ? 140 : 80
-
-    _ = self.projectLabel
-      |> UILabel.lens.font %~~ { _, label in
-        label.traitCollection.isRegularRegular
-          ? .ksr_title3()
-          : .ksr_headline(size: 14)
-      }
-      |> UILabel.lens.textColor .~ .ksr_text_navy_600
-
-    _ = self.projectNameContainerView
-      |> UIStackView.lens.layoutMargins .~ .init(top: Styles.grid(1), left: 0, bottom: 0, right: 0)
-      |> UIStackView.lens.layoutMarginsRelativeArrangement .~ true
-      |> UIStackView.lens.spacing .~ Styles.grid(1)
-
-    _ = self.separateView |> separatorStyle
-
-    _ = self.statsStackView |> UIStackView.lens.spacing .~ Styles.grid(1)
+    _ = self.progressStaticView
+      |> dropShadowStyleLarge()
+      |> UIView.lens.backgroundColor .~ .black
+      |> UIView.lens.alpha .~ 0.15
   }
 
   internal override func bindViewModel() {
     super.bindViewModel()
 
-    self.fundingTitleLabel.rac.attributedText = self.viewModel.outputs.fundingSmallLabelText
-    self.deadlineSubtitleLabel.rac.text = self.viewModel.outputs.deadlineSubtitleLabelText
-    self.deadlineTitleLabel.rac.text = self.viewModel.outputs.deadlineTitleLabelText
+    self.metadataBackgroundView.rac.backgroundColor = self.viewModel.outputs.progressBarColor
+    self.metadataLabel.rac.text = self.viewModel.outputs.metadataText
+    self.percentFundedLabel.rac.attributedText = self.viewModel.outputs.percentFundedText
+    self.progressBarView.rac.backgroundColor = self.viewModel.outputs.progressBarColor
     self.projectImageView.rac.imageUrl = self.viewModel.outputs.projectImageUrlMed
-    self.projectLabel.rac.text = self.viewModel.outputs.projectNameLabelText
+    self.projectLabel.rac.attributedText = self.viewModel.outputs.projectNameLabelText
+
+    self.viewModel.outputs.progress
+      .observeForUI()
+      .observeValues { [weak element = progressBarView] progress in
+        let anchorX = progress == 0 ? 0 : 0.5 / progress
+        element?.layer.anchorPoint = CGPoint(x: CGFloat(max(anchorX, 0.5)), y: 0.5)
+        element?.transform = CGAffineTransform(scaleX: CGFloat(min(progress, 1.0)), y: 1.0)
```

Comment:
Wondering if these should be separate outputs so that we can write tests for them? 🤔

Also, capturing of `progressBarView` in this way is fine but capturing `[weak self]` directly calling `self?.progressBarView` in the closure also works and I think is more in line with the rest of our code I think. Either is fine.